src/trace/trigger.ts

```diff
@@ -396,6 +396,7 @@ export function extractHTTPStatusCodeTag(
     if (typeof resultStatusCode === "number") {
       return resultStatusCode.toString();
     }
+    return resultStatusCode.toString();
```

Comment:
in this case it's already a string, so you can just return the status code itself.

Reply:
let's just always return `resultStatusCode.toString();` if it exists